### DISCUSSION TITLE
Ionic and Capacitor plugins removed?

### DISCUSSION THREAD

**Discussion by hrafnkellb-advania** [nxext/nx-extensions#1049] [Q&A] [answered]
Hey 😃 , I was just migrating my NX project to v17 and noticed that the documentation for Ionic and Capacitor disappeared.

https://nxext.dev/docs/ionic-react/getting-started.html
https://nxext.dev/docs/capacitor/getting-started.html

I also see that the following packages have been removed in this [commit](https://github.com/nxext/nx-extensions/commit/67c7419804174c9866337a74ebb78f7add96db00#diff-1f4a212f649e2075d56924cdc2a6150d26f9b8f7c20973d3bbc48a59a1230fc7)
@DominikPieper 

- capacitor
- ionic-angular
- ionic-react

I'm just a little bit confused since there is not mention of this in the docs.

Is there any plan to add Ionic/Capacitor support back, or has it been permanently removed?

UPDATE:

Just saw that the repository is being restructured, so the ionic related extensions are here
https://github.com/nxext/nx-extensions-ionic

**Comment by DominikPieper** [comment, marked as answer]
The ionic related packages are all together got their own repository. The docs need to get reintegrated but the plugins itself are just moved to https://github.com/nxext/nx-extensions-ionic

**Comment by sagarkm** [comment]
What about the npm plugins @nxext/capacitor? do we continue using the same? or that too would move into a different npm?
I see https://github.com/nxext/nx-extensions-ionic/pull/13 merged into dev - so the extension supports nx v17, but https://www.npmjs.com/package/@nxext/capacitor is still on v16.7.0 - last updated 3 months ago.

**Comment by DominikPieper** [comment]
@sagarkm They will stay the same. I just hadn't time to do the release for the Ionic packages

**Reply by sagarkm** [reply]
@DominikPieper Thanks for the wonderful work. It would be great if @nxext/capacitor supporting nx v17 could be published. That would unblock many of us :)

**Comment by DominikPieper** [comment]
@sagarkm done. I pursued the v17 versions

**Reply by sagarkm** [reply]
thanks @DominikPieper
